doc/rst/usingchapel/multilocale.rst

```diff
@@ -212,13 +213,24 @@ Troubleshooting
 +++++++++++++++
 
 If you are trying to debug job launch, try adding ``-v`` or
-``--dry-run`` to your program's command line and set:
+``--dry-run`` to your program's command line in order to
+see the command(s) that Chapel is executing on your behalf
+to get things running.
+
+For ``CHPL_COMM=gasnet`` runs, you may also want to consider setting
+one or both of:
 
   .. code-block:: bash
 
+    export GASNET_SPAWN_VERBOSE=1
     export GASNET_VERBOSEENV=1
 
-We've had best results with console I/O using:
+where the first prints more information about GASNet's job launch
+actions, and the second is used to audit environment variable
+settings.
+
+When running ``CHPL_COMM=gasnet`` programs, we've had best results
+with console I/O using:
```

Comment:
`GASNET_ROUTE_OUTPUT` only affects the behavior of udp-conduit (CHPL_COMM_SUBSTRATE=udp). 

So this paragraph might instead belong on your [udp-specific page](https://chapel-lang.org/docs/platforms/udp.html#using-udp)?

Reply:
@bonachea : Thanks for the suggestion.  I added it to the UDP page, but also retained it here, simply qualifying it as being udp-specific.